### PR TITLE
Create Progress bar indeterminate based on activity bar

### PR DESCRIPTION
**Pull request by korneliakobiela**
[Issue] #875
[Problem] Indeterminate bar didn't exist in the old version of TAU.
[Solution]
- Create a new example
- Change the styling element
- Change the animation type
- Add new graphics 

Signed-off-by: Kornelia Kobiela <korneliak.95@gmail.com>

### CODE REVIEW

**Comment by singa2000** [comment]
@korneliakobiela 
Would you update the code, please?

File: src/js/core/widget/core/progress/type/indeterminatebar.js [hunk 1]
```diff
@@ -38,43 +38,43 @@
...
-					activityBarElement = document.createElement("div");
+					indeterminateBarElement = document.createElement("div");
 
-					activityElement.classList.add(classes.uiActivitybar);
-					activityBarElement.classList.add(classes.uiActivitybarActivity);
+					indeterminateElement.classList.add(classes.uiIndeterminatebar);
+					indeterminateBarElement.classList.add(classes.uiIndeterminatebarActivity);
 
-					activityElement.appendChild(activityBarElement);
+					indeterminateBarElement.appendChild(indeterminateElement);
```
Comment (TomaszLukawskiSam) [inline src/js/core/widget/core/progress/type/indeterminatebar.js]: In this place you did probably wrong assign. 
`indeterminateElement.appendChild(indeterminateBarElement);`

File: src/css/profile/mobile/common/progress.less [hunk 2]
```diff
@@ -200,10 +197,10 @@ tau-progress {
 	mask-image: url('./images/controls/core_progress_pending_bg.svg');
```
Comment (TomaszLukawskiSam) [inline src/css/profile/mobile/common/progress.less]: obraz

**Comment by TomaszLukawskiSam** [comment]
Please fix commit message:
[Problem] <empty>

File: src/css/profile/mobile/common/progress.less [hunk 3]
```diff
@@ -176,60 +176,57 @@ tau-progress {
...
 	height: 100%;
 	padding: 0;
 	background-color: var(--progress-bar-color);
 	.transition(width 100ms);
-	mask-image: url('./images/controls/core_progress_pending_bg.svg');
-	.mask-repeat(repeat-x);
-	border-radius: 10 * @unit_base;
-	.animation(activitybar 1000ms infinite linear);
+	mask-image: url('./images/8_Progress/progress_bar_ind.png');
+	.mask-repeat(no-repeat);
```
Comment (TomaszLukawskiSam) [inline src/css/profile/mobile/common/progress.less]: you can replace these less methods by standard css methods. We have autoprefixer for it.

File: src/css/profile/mobile/common/progress.less (hunk 2 above)
Comment (korneliakobiela) [reply src/css/profile/mobile/common/progress.less]: Done!

**Comment by TomaszLukawskiSam** [comment]
@korneliakobiela 
You have to rebase and resolve conflicts

File: src/css/profile/mobile/common/progress.less [hunk 4]
```diff
@@ -176,62 +176,39 @@ tau-progress {
...
 	top: 0;
 	height: 100%;
 	padding: 0;
 	background-color: var(--progress-bar-color);
-	.transition(width 100ms);
-	mask-image: url('./images/controls/core_progress_pending_bg.svg');
-	.mask-repeat(repeat-x);
-	border-radius: 10 * @unit_base;
-	.animation(activitybar 1000ms infinite linear);
+	//transition: width 100ms;
```
Comment (TomaszLukawskiSam) [inline src/css/profile/mobile/common/progress.less]: remove or left this style instead of comment this line.

Comment (korneliakobiela) [reply src/css/profile/mobile/common/progress.less]: Done!

File: src/css/profile/mobile/common/progress.less (hunk 3 above)
Comment (korneliakobiela) [reply src/css/profile/mobile/common/progress.less]: Done!

**Comment by korneliakobiela** [comment]
The guideline doesn't contain information about animation parameters. I created Issue for that. https://github.com/Samsung/TAU/issues/1037

**Comment by TomaszLukawskiSam** [comment]
What's going on with CI?

**Comment by lslachciak** [comment]
Looks like CI passed. @TomaszLukawskiSam could you make final review?

**Review by TomaszLukawskiSam — APPROVED** [review]
Looks good to me.

**Pull request merged**